web/js/codeworld_shared.js

```diff
@@ -1559,78 +1559,84 @@ function preFormatMessage(msg) {
 }
 
 function printMessage(type, message) {
-  const outputDiv = document.getElementById('message');
+  const $outputBlock = $('#message');
```

Comment:
Why all the '$' prefixes for variables?

Reply:
This prefix indicates that given DOM element was spawned (or selected) by jQuery and has all associated methods attached to it. It's fairly common to do so, but if you don't like it, I'll remove it.